File: c++/1005.cpp
//El problema no se pudo resolver pero esto es lo que se tiene
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    cin>>n;
    
    for(int i=0; i<n; i++){
        
        string cadena;
        cin>>cadena;
        
        int
        con = 0;
        
        float v[5];
        for(int j =0; j < 5; j++){
            v[j]=0;
        }
        for(char x: cadena){
            con++;
            
            switch(x) 
            {
                case 'a': 
                v[0]++;
                break;
                case 'e':
                v[1]++;
                break;
                case 'i': 
                v[2]++;
                break;
                case 'o': 
                v[3]++;
                break;
                case 'u': 
                v[4]++;
                break; 
            }
        }
        cout<<"Caso "<<(i+1)<<":\n";
            printf("a= %.2f\n",((v[0]/con)*100));
            printf("e= %.2f\n",((v[1]/con)*100));
            printf("i= %.2f\n",((v[2]/con)*100));
            printf("o= %.2f\n",((v[3]/con)*100));
            printf("u= %.2f\n",((v[4]/con)*100));
    }
    return 0;
}

File: python/1005.py
#Sin resolver :c
for i in range(int(input())):
    vector = [0.00 for i in range(5)]
    length = 0
    for j in input():
        length+=1
        if j=='a':vector[0]+=1
        elif j=='e':vector[1]+=1
        elif j=='i':vector[2]+=1
        elif j=='o':vector[3]+=1
        elif j=='u':vector[4]+=1
    print(f"Caso {i+1}:\na= {round((vector[0]/length*100),2):.2f}\ne= {round((vector[1]/length*100),2):.2f}\ni= {round((vector[2]/length*100),2):.2f}\no= {round((vector[3]/length*100),2):.2f}\nu= {round((vector[4]/length*100),2):.2f}")